### ISSUE TITLE
Blur not working android

### ISSUE THREAD

**Issue by monkhsaikhan** [react-native-community/react-native-blur#322]
Blur working on ios perfect. But not working on android

here is my code

`<BlurView
                        style={{position: "absolute", top: 70, left: 0, bottom: 0, right: 0}}
                        blurType="light"
                        blurAmount={5}
                        blurRadius={15}
                        downsampleFactor={5}
                        overlayColor={'rgba(255, 255, 255, .25)'}
                        viewRef={this.state.viewRef}
                    /> :
                    <BlurView
                        style={{position: "absolute", top: 70, left: 0, bottom: 0, right: 0}}
                        blurType="light"
                        blurAmount={5}
                    /> 
                    ) : null`

**Comment by xclidongbo** [comment]
some to me.

**Comment by boygiandi** [comment]
me too :( Please fix it

**Comment by athirah-yahya** [comment]
Any follow up on this? I am using RN 0.60.4 here.

No error nor warning

**Comment by boygiandi** [comment]
yes. You can use this. Seem like to work in normal situation. https://github.com/lvlrSajjad/react-native-blur-overlay
But when I try to blur a video view, it doesn't work

**Comment by MrFreemind** [comment]
Try to add some background-color to view that you need to blur. I've spent so much time to figure out why it's not working on android, but that helped me out. Interesting that in this case blur depends on background opacity. First I've added transparent color and it start working but with some bugs until I added just #fff color.

**Comment by rhamag** [comment]
Had the same issue. These two things worked for me:

Make sure you have a backgroundColor on the container you want to blur. Make sure the component rerenders:
```
style={{ backgroundColor: 'transparent' }}
ref={n => {
  if (n && viewRef === null) {
    setViewRef(findNodeHandle(n))
  }
}}
```

**Comment by xhirazi** [comment]
+1

**Comment by anisfrontmen** [comment]
+1

**Comment by MrFreemind** [comment]
@xhirazi @anisfrontmen check out this comment. It might be helpful.
https://github.com/react-native-community/react-native-blur/issues/328#issuecomment-575678258

**Comment by Psiiirus** [comment]
Updated to 3.5(dev-master) and this seems to be fixed...i guess its because android is using a totally different android-lib to blur since 3.5

**Comment by maheshmuttinti** [comment]
> 

add `zIndex: 100` to BlurView style. it will resolve error.
